<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>743</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>754</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AldoCAD - The best of</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Objects in this world</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomPlus">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>330</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom +</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomMinus">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>360</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom -</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="objectsTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="translate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scale">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cohenSutherlandClipping">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cohen-Sutherland line clipping</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="liangBarskyClipping">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Liang-Barsky line clipping</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>10</y>
      <width>470</width>
      <height>490</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Graphic world view</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="worldView">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>27</y>
       <width>450</width>
       <height>380</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="interactive">
      <bool>false</bool>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>430.000000000000000</width>
       <height>360.000000000000000</height>
      </rectf>
     </property>
     <property name="transformationAnchor">
      <enum>QGraphicsView::NoAnchor</enum>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::MinimalViewportUpdate</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateWorld">
     <property name="geometry">
      <rect>
       <x>373</x>
       <y>407</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="worldDegrees">
     <property name="geometry">
      <rect>
       <x>282</x>
       <y>411</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>411</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate window</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>411</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>degrees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="center">
     <property name="geometry">
      <rect>
       <x>373</x>
       <y>433</y>
       <width>90</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="parallel">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>410</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parallel projection</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="perspective">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>434</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Perspective projection</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dop">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>458</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>135</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>461</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance of projection</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_file"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuObjects">
    <property name="title">
     <string>Objects</string>
    </property>
    <addaction name="actionClear_world"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_point"/>
    <addaction name="actionAdd_line"/>
    <addaction name="actionAdd_polygon"/>
    <addaction name="actionAdd_BSpline"/>
    <addaction name="actionAdd_object_3D"/>
    <addaction name="actionAdd_surface"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuObjects"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClear_world">
   <property name="text">
    <string>Clear world</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionAdd_point">
   <property name="text">
    <string>Add point</string>
   </property>
  </action>
  <action name="actionAdd_line">
   <property name="text">
    <string>Add line</string>
   </property>
  </action>
  <action name="actionAdd_polygon">
   <property name="text">
    <string>Add polygon</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_BSpline">
   <property name="text">
    <string>Add B-Spline</string>
   </property>
  </action>
  <action name="actionAdd_object_3D">
   <property name="text">
    <string>Add object 3D</string>
   </property>
  </action>
  <action name="actionAdd_surface">
   <property name="text">
    <string>Add amazing surface</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
